ofrak_core/pytest_ofrak/elf/assets/Makefile

```diff
@@ -1,5 +1,5 @@
-
-CC=gcc
+# Specifying the full name is necessary for some tests to work on ARM
+CC=x86_64-linux-gnu-gcc
```

Comment:
In my opinion, this is not a good change, because the premise is faulty. We should not be building these ELF files dynamically in the first place. They should instead be Git LFS committed to the repo like all of our other test assets. 

We don't gain anything by building them dynamically except for requiring additional dependencies to run the tests.